### ISSUE TITLE
Make build scan URLs clickable in warning and notices

### ISSUE THREAD

**Issue by bigdaz** [gradle/gradle-build-action#102]
When a Gradle invocation generates a build scan, a notice is added with the build scan URL. It would be great if this was clickable.

It's unclear if this is possible, or if it requires a change to the GitHub actions infrastructure.

**Comment by bigdaz** [comment]
I've spent quite some time trying to find a way to make this work, to no avail. If anyone has knowledge about how to make clickable links in a notice emitted from GitHub Actions, I'd love to hear it.

I also investigated creating a separate [check-run](https://docs.github.com/en/rest/reference/checks#check-runs) for each Gradle invocation: this would allow the Gradle invocations to be listed separately from the workflow jobs, with rich markdown content associated.  Unfortunately this doesn't work well when you have more than 1 workflow configured: [the `check-run` gets randomly assigned to one of the workflows](https://github.community/t/github-actions-status-checks-created-on-incorrect-check-suite-id/16685#M2466), often the wrong one.
